web/styles/app_variables.css

```diff
@@ -408,6 +408,9 @@
     --opacity-sidebar-heading-hover: 0.9;
     --opacity-left-sidebar-muted: 0.55;
     --opacity-left-sidebar-muted-hover: 0.75;
+    /* Since the default opacity for the mentions icon
+       is too close to --opacity-left-sidebar-muted */
```

Comment:
Can we reference the variable name for opacity of mentions as well in this comment?

(Or if it's hardcoded, can we extract a variable for that?)